### PR TITLE
Link users to Multiline strings section

### LINKED ISSUE #21529: Clarify how multiline output values can be set via `GITHUB_OUTPUT`
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter

### What part(s) of the article would you like to see updated?

Due to recent changes, I had to switch our codeflows from using `set-output` to the method described in the section on "Setting an output parameter", by appending to (the file with the name in the variable) `$GITHUB_OUTPUT`.

This is very similar to how `$GITHUB_ENV` works in the previous section.

I had need to post a multiline output, and so decided to just try the technique described for "Environment files" (the preceding section). That is, I used the pattern
```
{name}<<{delimiter}
{value}
{delimiter}
```
and ended up with code looking like this:
```yaml
steps:
  - name: Set the output in bash
    id: step_one
    run: |
      echo 'JSON_RESPONSE<<EOF' >> $GITHUB_OUTPUT
      curl https://example.lab >> $GITHUB_OUTPUT
      echo 'EOF' >> $GITHUB_OUTPUT
```

And as far as I can tell, this worked great (and is much simpler than what I had to do before).

I think it would be beneficial if this possibility was pointed out by the documentation. Simply saying something akin to "multi-line output values can be set with the same technique used in the previous section to define multi-line environment variables" would already help a lot.

### Additional information

_No response_

---
edited by maintainer

### Content plan [here](https://github.com/github/docs/issues/21529#issuecomment-1289693089)

### PR DESCRIPTION
**Pull request by dawei-wang**
Fixes github/docs#21529

### Why:

Closes github/docs#21529

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@dawei-wang Thanks for submitting a PR with this fix! I'll get this triaged for review ⚡

File: content/actions/using-workflows/workflow-commands-for-github-actions.md [hunk 1]
```diff
@@ -733,7 +733,7 @@ steps:
 {% ifversion actions-save-state-set-output-envs %}
 ## Setting an output parameter
 
-Sets a step's output parameter. Note that the step will need an `id` to be defined to later retrieve the output value.
+Sets a step's output parameter. Note that the step will need an `id` to be defined to later retrieve the output value. Multi-line output values can be set with the same technique used in the [Multiline strings](/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings) section to define multi-line environment variables.
```
Comment (mchammer01) [inline content/actions/using-workflows/workflow-commands-for-github-actions.md:736]: Used double quotes for the link and the active voice (to be in line with our content model)
```suggestion
Sets a step's output parameter. Note that the step will need an `id` to be defined to later retrieve the output value. You can set multi-line output values with the same technique used in the "[Multiline strings](/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings)" section to define multi-line environment variables.
```

**Review by mchammer01 — DISMISSED** [review]
@dawei-wang 👋🏻 - thanks for your contribution, it LGTM ✨ 
I made a couple of minor suggestions that I am going to commit.
In [this comment](https://github.com/github/docs/issues/21529#issuecomment-1289693089), @skedwards88 suggested we included an example.

**Review by mchammer01 — COMMENTED** [review]
@dawei-wang 👋🏻 - thanks for your contribution, it LGTM ✨ 
I made a couple of minor suggestions that I am going to commit.
In [this comment](https://github.com/github/docs/issues/21529#issuecomment-1289693089), @skedwards88 suggested we included an example.
What do you think? Are you happy to add the example in your PR?

**Comment by mchammer01** [comment]
Closing this for 5 mins or so as the Deploy Preview Environment test seems to be unhappy.

**Comment by cmwilson21** [comment]
@dawei-wang 👋 - Just checking in. Have you had a chance to think about adding an example per the request [here](https://github.com/github/docs/pull/23027#pullrequestreview-1287408081)?

**Review by cmwilson21 — APPROVED** [review]

**Comment by 1oglop1** [comment]
@cmwilson21 why is this merged? I've just spent several hours trying to get it working without success

```
steps:
    - id: submit
      name: submit deploy
      shell: bash
      env:
        GITHUB_TOKEN: "${{ inputs.gh_token }}"
        PULUMI_STACK: ${{inputs.app_environment}}
        APP: ${{inputs.app_name || inputs.app_environment }}
        REF: ${{ inputs.branch || github.ref }}
      run: |
        response=$(curl --silent --show-error --fail \
          -X POST "https://api.github.com/repos/$GITHUB_REPOSITORY/deployments" \
          -H "Authorization: token $GITHUB_TOKEN" \
          -H "Content-Type: text/json; charset=utf-8" \
          -d @- <<EOF
        {
          "ref":"$REF",
          "description": "autodeploy",
          "required_contexts": [],
          "auto_merge": false,
          "environment": "$APPENV",
          "payload": {
            "environment": "$APP",
            "pulumi_stack": "$PULUMI_STACK",
            "app": "$APP"
          }
        }
        EOF
        )

        echo "response<<EOF" >> $GITHUB_OUTPUT
        echo "$response" >> $GITHUB_OUTPUT
        echo "EOF" >> $GITHUB_OUTPUT
```

with error
```
echo {
/home/runner/work/_temp/343c5f31-a8d6-4aba-92cf-92b61da19f67.sh: line 2: url:: command not found
{
Error: Process completed with exit code 127.
```

**Pull request merged**